### PR TITLE
remove  g2pk from Korean G2P 

### PR DESCRIPTION
**Pull request by eunjung31**
Removing [g2pK](https://github.com/Kyubyong/g2pK) dependency for Korean G2P

### CODE REVIEW

File: epitran/data/post/kor-Hang.txt [hunk 1]
```diff
@@ -1,42 +1,84 @@
...
+::jvowel:: = i|je|jɛ|ja|jo|ju|jʌ
 
-% remove ㅇ if followed by a vowel
+% remove ㅇ if followed by a vowel (if it is in initial consonant position)
 ŋ -> 0 / _ (::vowels::)
 
-% change to medial allophone if surrounded by vowels
-k -> g / (::vowels::) _ (::vowels::)
-t -> d / (::vowels::) _ (::vowels::)
-t͡ɕ -> dʑ / (::vowels::) _ (::vowels::)
```
Comment (dmort27) [inline epitran/data/post/kor-Hang.txt:11]: there should be a ligature tie on dʑ (that is, it should be d͡ʑ)

File: epitran/data/post/kor-Hang.txt [hunk 2]
```diff
@@ -1,42 +1,84 @@
...
+nhk -> nkʰ / (::vowels::) _ (::vowels::)
+nht͡ɕ -> nt͡ɕʰ / (::vowels::) _ (::vowels::)
+lht -> lhtʰ / (::vowels::) _ (::vowels::)
+lhk -> lhkʰ / (::vowels::) _ (::vowels::)
+lht͡ɕ -> lht͡ɕʰ / (::vowels::) _ (::vowels::)
+lph -> lpʰ / (::vowels::) _ (::vowels::)
+lkh -> lkʰ / (::vowels::) _ (::vowels::)
+nt͡ɕh -> nt͡ɕʰ / (::vowels::) _ (::vowels::)
 
-% change to final allophone if followed by an initialconsonant
```
Comment (dmort27) [inline epitran/data/post/kor-Hang.txt:16]: initialconsonant → initial consonant

File: epitran/data/post/kor-Hang.txt [hunk 3]
```diff
@@ -1,42 +1,84 @@
...
-k̚s͈ -> k̚ / _ (::initialconsonants::|#)
-lɡ -> k̚ / _ (::initialconsonants::|#)
-ndʑ -> n / _ (::initialconsonants::|#)
+% Substitution of lax to tensce for consonant clusters with 's'
+ks -> ks͈ / (::vowels::) _ (::vowels::)
+ls -> ls͈ / (::vowels::) _ (::vowels::)
+ps-> ps͈ / (::vowels::) _ (::vowels::)
+
+% Simplification of consonant clusters to single consonants at coda
```
Comment (dmort27) [inline epitran/data/post/kor-Hang.txt:30]: Shouldn't these also apply word-finally?

File: epitran/simple.py [hunk 4]
```diff
@@ -61,8 +61,8 @@ def __init__(self, code: str, preproc: bool = True, postproc: bool = True, ligat
 
         self.nils = defaultdict(int)
         
-        if code == "kor-Hang":
-            self.hang_g2p = G2p()
+        # if code == "kor-Hang":
+        #     sellsf.hang_g2p = G2p()
```
Comment (dmort27) [inline epitran/simple.py:65]: `sellsf` looks like an error

Comment (dmort27) [reply epitran/simple.py:65]: On, never mind—commented out.

**Review by dmort27 — CHANGES_REQUESTED** [review]
This is mostly good, but there are a few small points I have commented on. Fix these and then I will merge the PR.

File: epitran/data/post/kor-Hang.txt (hunk 3 above)
Comment (eunjung31) [reply epitran/data/post/kor-Hang.txt:30]: The second consonant in a coda cluster shifts to the next syllable if followed by a vowel, except /s/, which becomes /s͈/. I'll update the annotation descriptions to clarify this rule.

**Review by eunjung31 — COMMENTED** [review]
Responded to all comments!

**Pull request merged**